### PR TITLE
openfoam: delete test print and split long lines.

### PR DESCRIPTION
**Pull request by tkameyama**

### CODE REVIEW

**Comment by adamjstewart** [comment]
@olesenm can you review?

File: var/spack/repos/builtin/packages/openfoam/package.py [hunk 1]
```diff
@@ -915,7 +914,9 @@ def __init__(self, spec, **kwargs):
         # Build WM_OPTIONS
         # ----
         # WM_LABEL_OPTION=Int$WM_LABEL_SIZE
-        # WM_OPTIONS=$WM_ARCH$WM_COMPILER$WM_PRECISION_OPTION$WM_LABEL_OPTION$WM_COMPILE_OPTION
+        # WM_OPTIONS=$WM_ARCH \
```
Comment (olesenm) [inline var/spack/repos/builtin/packages/openfoam/package.py]: This is misleading, there are no spaces in the resulting variable

File: var/spack/repos/builtin/packages/openfoam/package.py [hunk 2]
```diff
@@ -584,10 +583,10 @@ def make_fujitsu_rules(self):
                     backup=False, string=True)
 
     def configure(self, spec, prefix):
-        """Make adjustments to the OpenFOAM configuration files in their various
-        locations: etc/bashrc, etc/config.sh/FEATURE and customizations that
-        don't properly fit get placed in the etc/prefs.sh file (similiarly for
-        csh).
+        """Make adjustments to the OpenFOAM configuration files in their
```
Comment (olesenm) [inline var/spack/repos/builtin/packages/openfoam/package.py:586]: Fine either way

File: var/spack/repos/builtin/packages/openfoam/package.py (hunk 1 above)
Comment (tkameyama) [reply var/spack/repos/builtin/packages/openfoam/package.py]: fixed.

**Review by alalazo — APPROVED** [review]
This just seem to remove a print to screen and adjust a few docstring comments, so I'm going to merge it.

**Pull request merged**